### PR TITLE
More createFromTransaction tests

### PR DESCRIPTION
**Pull request by hugo-dc**

### CODE REVIEW

**Comment by lrettig** [comment]
Looks great, @hugo-dc what's the status of this? Ready for review?

**Comment by hugo-dc** [comment]
@lrettig still have to write two more test cases in order to close https://github.com/ewasm/tests/issues/15

**Comment by hugo-dc** [comment]
@lrettig I added two more test cases, please review

File: src/GeneralStateTestsFiller/stEWASMTests/createFromTransactionExceedGasLimitFiller.yml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    - indexes:
+        data: !!int -1
+        gas: !!int -1
+        value: !!int -1
+      network:
+        - ALL
+      result:
+        a94f5374fce5edbc8e2a8697c15331677e6ebf0b:
+          balance: '99995025681'
+        # should not create contract, exceeds max code size
```
Comment (lrettig) [inline src/GeneralStateTestsFiller/stEWASMTests/createFromTransactionExceedGasLimitFiller.yml]: Typo in this comment

File: src/GeneralStateTestsFiller/stEWASMTests/createFromTransactionExceedGasLimitFiller.yml [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+        (export "memory" (memory 0))
+        (export "main" (func $main))
+        (func $main
+          (call $return
+            (i32.const 0)
+            (i32.const 24576))
+        )
+      )
+    gasLimit:
+    - '0x4be6ef'
```
Comment (lrettig) [inline src/GeneralStateTestsFiller/stEWASMTests/createFromTransactionExceedGasLimitFiller.yml:50]: Could you include slightly more on the gas math here? What's the gas cost for this tx and how is it calculated?

**Review by lrettig — CHANGES_REQUESTED** [review]
Added some comments. What is `createFromTransactionReturnRandomBytes` testing, exactly? That you can create a contract from a tx using random bytes? I don't completely understand.

File: src/GeneralStateTestsFiller/stEWASMTests/createFromTransactionExceedGasLimitFiller.yml (hunk 2 above)
Comment (hugo-dc) [reply src/GeneralStateTestsFiller/stEWASMTests/createFromTransactionExceedGasLimitFiller.yml:50]: After I wrote the first createFromTransaction test case @axic suggested more scenarios that we could tests, Returning Zero bytes, returning two bytes and random bytes.

Comment (hugo-dc) [reply src/GeneralStateTestsFiller/stEWASMTests/createFromTransactionExceedGasLimitFiller.yml:50]: @lrettig I fixed the comments, also added some more to show the contract creation is exceeding the gas limit. Actually I had trouble calculating the gas usage in this test case, I couldn't calculate why it needs 4974320 (0x4be6f0) gas to create the contract, but the test shows that if the gas limit is less than that (4974319 / 0x4be6ef), the creation fails.

**Review by lrettig — APPROVED** [review]

**Comment by lrettig** [comment]
LGTM thanks @hugo-dc !
